apps/server/src/modules/tldraw/repo/y-mongodb.repo.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { EntityManager } from '@mikro-orm/mongodb';
+import { Test, TestingModule } from '@nestjs/testing';
+import { cleanupCollections } from '@shared/testing';
+import { MongoMemoryDatabaseModule } from '@infra/database';
+import { ConfigModule } from '@nestjs/config';
+import { LegacyLogger } from '@src/core/logger';
+import { createMock } from '@golevelup/ts-jest';
+import * as Yjs from 'yjs';
+import { tldrawEntityFactory } from '../factory';
+import { TldrawDrawing } from '../entities';
+import { TldrawWs } from '../controller';
+import { TldrawWsService } from '../service';
+import { TldrawBoardRepo } from './tldraw-board.repo';
+import { TldrawRepo } from './tldraw.repo';
+import { YMongodb } from './y-mongodb';
+
+jest.mock('yjs', () => {
+	const moduleMock: unknown = {
+		__esModule: true,
+		...jest.requireActual('yjs'),
+	};
+	return moduleMock;
+});
+
+describe('YMongoDb', () => {
+	let testingModule: TestingModule;
+	let mdb: YMongodb;
+	let em: EntityManager;
+
+	beforeAll(async () => {
+		testingModule = await Test.createTestingModule({
+			imports: [
+				MongoMemoryDatabaseModule.forRoot({ entities: [TldrawDrawing] }),
+				ConfigModule.forRoot({
```

Comment:
It can be simplify. Check the config mocking with @golevelup/ts-jest on other places and should be done in setup of the test.

Reply:
here we cannot use deepmocked as we have to check if entities in database